<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewPort</class>
 <widget class="QDialog" name="NewPort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="gb_AddNewPort">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Add New Port</string>
   </property>
   <widget class="QGroupBox" name="gb_Commodities">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>221</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Commodity Prices</string>
    </property>
    <widget class="QLineEdit" name="le_Leather">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>240</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_Wine">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="l_Leather">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>244</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Leather:</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_Cloth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>65</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cloth:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_Minerals">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="l_Wine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>126</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Wine:</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_Food">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>34</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Food:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_Grain">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="l_Minerals">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>95</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Minerals:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_Timber">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_Paper">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_Food">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="l_Grain">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>154</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Grain:</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_Timber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>185</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Timber:</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_Paper">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>214</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Paper:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_Cloth">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>275</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="le_Gun_Training">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>151</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>N</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="l_Latitude">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Latitude:</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_Gun_Training">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>155</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Gun_Training:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_Longitude">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>79</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="l_Longitutde">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>82</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Longitude:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_Latitude">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>116</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_PortName">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="l_PortName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>14</y>
       <width>80</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Port Name:</string>
     </property>
    </widget>
    <zorder>l_Latitude</zorder>
    <zorder>l_Gun_Training</zorder>
    <zorder>le_Longitude</zorder>
    <zorder>l_Longitutde</zorder>
    <zorder>le_Latitude</zorder>
    <zorder>le_PortName</zorder>
    <zorder>l_PortName</zorder>
    <zorder>le_Gun_Training</zorder>
   </widget>
   <zorder>gb_Commodities</zorder>
   <zorder>groupBox</zorder>
   <zorder>buttonBox</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_PortName</tabstop>
  <tabstop>le_Longitude</tabstop>
  <tabstop>le_Latitude</tabstop>
  <tabstop>le_Gun_Training</tabstop>
  <tabstop>le_Food</tabstop>
  <tabstop>le_Cloth</tabstop>
  <tabstop>le_Minerals</tabstop>
  <tabstop>le_Wine</tabstop>
  <tabstop>le_Grain</tabstop>
  <tabstop>le_Timber</tabstop>
  <tabstop>le_Paper</tabstop>
  <tabstop>le_Leather</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewPort</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewPort</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
